appengine/gophers/gophers-5/index.html

```diff
@@ -0,0 +1,113 @@
+<!doctype html>
+<html>
+<head>
+  <title>The Gopher Network</title>
+
+  <link type="text/css" rel="stylesheet" href="/static/style.css" />
+
+  <!-- See https://github.com/firebase/firebaseui-web. -->
+  <!-- [START init_firebase] -->
+  <!-- *******************************************************************************************
+       * TODO(DEVELOPER): Paste the initialization snippet from:
+       * Firebase Console > Overview > Add Firebase to your web app.
+       ***************************************************************************************** -->
+  <!-- [END init_firebase] -->
+
+  <!-- [START include_firebaseui] -->
+  <script src="https://cdn.firebase.com/libs/firebaseui/2.6.2/firebaseui.js"></script>
+  <link type="text/css" rel="stylesheet" href="https://cdn.firebase.com/libs/firebaseui/2.6.2/firebaseui.css" />
+  <!-- [START include_firebaseui] -->
+  <!-- [START handle_login] -->
+  <script>
```

Comment:
might want to move the script into a js file.

Reply:
I'm on the fence about this. On one hand, it's nice to have everything visible in one file (since there are two places to edit/look at). On the other hand, it's a bit long. So, a separate file makes more sense.

Reply:
Moved to a standalone file.